os-3.10/scripts/node01.sh

```diff
@@ -74,3 +74,29 @@ done
 
 /usr/bin/oc create -f /tmp/local-volume.yaml
 
+# enable the CPU manager on compute nodes, besides master
+# updating it on the master as nodes cannot update comfigmaps
+
+sed -i 's/kubeletArguments:/kubeletArguments:\n  cpu-manager-policy:\n  - static\n  system-reserved:\n  - cpu=500m\n  kube-reserved:\n  - cpu=500m/' /etc/origin/node/node-config.yaml
+sed -i 's/feature-gates:/feature-gates:\n  - CPUManager=true/' /etc/origin/node/node-config.yaml
+
+/usr/bin/oc create cm node-config-compute -n openshift-node --from-file=/etc/origin/node/node-config.yaml -o yaml --dry-run |oc replace -f -
+
+cat >post_deployment_cpu_manager_config <<EOF
+- hosts: nodes, new_nodes
+  tasks:
+    - name: Clean cpu manager state
+      block:
+        - file:
+            state: absent
+            path: /var/lib/origin/openshift.local.volumes/cpu_manager_state
+        - service:
+            name: origin-node
+            state: restarted
+            enabled: yes
+EOF
+
+sleep 200
```

Comment:
Why do you need sleep here?

Reply:
I wish I didn't have to.
openshift runs a script[1] on its start and run it in the background.
It's basically generates an updated config from a configmap and restarts kubelet.
Overall, if it doesn't succeed it sleeps 180 - no matter what I did I couldn't enable the cpu manager
before sleep 200.

I will open a bug for origin and it's ansible so they will allow us to start the cpu manager in a more efficient way. 

[1] https://paste.fedoraproject.org/paste/sAPpMV0XJDlBkKMKqVvfrA